packages/devtools_app/lib/src/flutter/split.dart

```diff
@@ -254,20 +255,23 @@ class _SplitState extends State<Split> {
           child: widget.children[i],
         ),
         if (i < widget.children.length - 1)
-          GestureDetector(
-            key: widget.dividerKey(i),
-            behavior: HitTestBehavior.translucent,
-            onHorizontalDragUpdate: (details) =>
-                isHorizontal ? updateSpacing(details, i) : null,
-            onVerticalDragUpdate: (details) =>
-                isHorizontal ? null : updateSpacing(details, i),
-            // DartStartBehavior.down is needed to keep the mouse pointer stuck to
-            // the drag bar. There still appears to be a few frame lag before the
-            // drag action triggers which is't ideal but isn't a launch blocker.
-            dragStartBehavior: DragStartBehavior.down,
-            child: widget.splitters != null
-                ? widget.splitters[i]
-                : _defaultSplitter(layoutHeight: height, layoutWidth: width),
+          MouseRegion(
+            cursor: SystemMouseCursors.grab,
```

Comment:
FYI, there are now resize cursors that might work better - https://master-api.flutter.dev/flutter/rendering/SystemMouseCursors/resizeLeftRight-constant.html and https://master-api.flutter.dev/flutter/rendering/SystemMouseCursors/resizeUpDown-constant.html

Reply:
👍 good to know; I'll open an issue to track

Reply:
https://github.com/flutter/devtools/issues/2183